extensions/3DTILES_tile_metadata/README.md

```diff
@@ -0,0 +1,161 @@
+# 3DTILES_tile_metadata
+
+## Overview
+
+This extension to 3D Tiles allows for storage of per-tile metadata in external binary buffers.
+
+## Indexing
+
+When using 3D Tiles 1.0, the tile metadata is indexed using the Depth First Traversal of the `tileset.json`.
+
+When using the `3DTILES_implicit_tiling` extension, the tile metadata is indexed using the metadata bitstream.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_tile_metadata Tileset JSON extension
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**properties**|`object`|An object containing one or more metadata properties associated with each tile.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
+
+---------------------------------------
+### 3DTILES_tile_metadata.properties
+
+This property of the 3DTILES_tile_metadata object enumerates the different metadata properties associated with each tile in the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**semantic**|`string`|The name of the feature. Semantic names must be unique.|No|
+|**bufferView**|`integer`|The index of the bufferView.|☑️ Yes|
+
+
+Application-specific semantics may also be defined, with the caveat that they must begin with an underscore, e.g. _CLASSIFICATION.
+
+---------------------------------------
+### 3DTILES_tile_metadata.bufferViews
+
+`bufferViews` provide a typed view into a `buffer`.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**buffer**|`integer`|The index of the buffer.|☑️ Yes|
+|**byteOffset**|`integer`|The offset relative to the start of the buffer in bytes.|☑️ Yes|
+|**byteLength**|`integer`|The length of the bufferView in bytes.| ☑️ Yes|
+|**elementByteOffsetsBufferView**|`integer`|The index of the bufferView containing byte offsets for each element. Must be defined for the STRING type.|No|
+|**type**|`string`|Specifies if the attribute is a scalar, vector, matrix or string.|No, default is one element.|
+|**componentType**|`string`|The datatype of components in the attribute.|☑️ Yes|
+
+Allowed `componentType`s:
+
+- `"BYTE"`
+- `"UNSIGNED_BYTE"`
+- `"SHORT"`
+- `"UNSIGNED_SHORT"`
+- `"INT"`
+- `"UNSIGNED_INT"`
+- `"FLOAT"`
+- `"DOUBLE"`
+
+Allowed `type`s:
+
+| `type` | Number of components |
+|:------:|:--------------------:|
+| `"SCALAR"` | 1 |
+| `"STRING"` | 1 |
```

Comment:
Should be stated somewhere that these are UTF-8 strings

Reply:
Added a note below the list of element types in `3DTILES_binary_buffers`.